### PR TITLE
fix: some snapshot requests failed

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes mf-2342

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/hooks/useVotes.ts [hunk 1]
```diff
@@ -49,12 +38,11 @@ async function Suspender(id: ProposalIdentifier['id'], space: ProposalIdentifier
                     : undefined,
                 address: v.voter,
                 authorIpfsHash: v.ipfs ?? v.id,
-                balance: sumBy(scores, (score) => score[v.voter.toLowerCase()] ?? 0),
-                scores: strategies.map((_strategy, i) => scores[i][v.voter] || 0),
+                balance: v.vp,
+                scores: [],
```
Comment (hanszhoou) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/hooks/useVotes.ts]: Here `v.vp` can be assigned to `balance`, and `scores` is useless for `VotesCard`.

But please double check
`packages/mask/src/plugins/Snapshot/SNSAdaptor/VotingCard.tsx`, 
`packages/mask/src/plugins/Snapshot/SNSAdaptor/hooks/usePower.ts`,
The voting power is derived from the `scores`, if you can find something to generate the voting power, then `PluginSnapshotRPC.getScores` can be removed.

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/hooks/useVotes.ts [hunk 2]
```diff
@@ -49,12 +38,11 @@ async function Suspender(id: ProposalIdentifier['id'], space: ProposalIdentifier
...
                 strategySymbol: proposal.space.symbol ?? strategies[0].params.symbol,
                 timestamp: v.created,
             }
         })
-        .sort((a, b) => (isSameAddress(a.address, account) ? -1 : b.balance - a.balance))
```
Comment (hanszhoou) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/hooks/useVotes.ts]: This sorting should not be removed.

File: packages/mask/src/plugins/Trader/SNSAdaptor/index.tsx [hunk 3]
```diff
@@ -40,6 +40,8 @@ const sns: Plugin.SNSAdaptor.Definition<
             Content({ result: _resultList, isProfilePage }) {
                 const { Others } = useWeb3State(NetworkPluginID.PLUGIN_EVM)
                 const resultList = _resultList as Web3Helper.TokenResultAll[]
+                console.log('---------------------------')
+                console.log(resultList)
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Trader/SNSAdaptor/index.tsx]:
```suggestion
```

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/hooks/useVotes.ts [hunk 4]
```diff
@@ -1,26 +1,31 @@
...
+    }
+
+    return scores
+}
+
 async function Suspender(id: ProposalIdentifier['id'], space: ProposalIdentifier['space'], account?: string) {
     const proposal = useProposal(id)
-
-    const voters = proposal.votes.map((v) => v.voter)
-    const scores = await PluginSnapshotRPC.getScores(
```
Comment (hanszhoou) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/hooks/useVotes.ts:16]: Please remove the outdated `PluginSnapshotRPC.getScores()` at `packages/mask/src/plugins/Snapshot/SNSAdaptor/hooks/usePower.ts` as well.

**Review by hanszhoou — APPROVED** [review]

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**